.github/workflows/hw-bulk-dep-upgrades-with-dispatch.yml

```diff
@@ -0,0 +1,15 @@
+name: Upgrade dependencies (dispatch variant)
+on:
+  workflow_dispatch:
+
+jobs:
+  upgrade:
+    # using `main` as the ref will keep your workflow up-to-date
+    uses: hashicorp/vault-workflows-common/.github/workflows/bulk-dependency-updates.yaml@add-update-plugin-check-workflow
```

Comment:
this workflow is the same as the other one, with a few changes for prototype reasons:

1) this references the hackweek branch of the workflows
2) I cut the cron trigger
3) The repository is just the repo name, with the owner cut.

This last change is a bit unfortunate, but it's because we wrote the vault-side workflow as though it was just the repo name, and changing that is complicated by the fact that `workflow_dispach` workflows need to be on `main` to be seen/run.